src/helpers/configuration.sh

```diff
@@ -24,7 +24,7 @@ fi
 if [[ "$(is_git_town_configured)" == false ]]; then
   echo "Git Town hasn't been configured for this repository."
   echo "Please run 'git town config --setup'."
-  echo -n "Would you like to do that now? [y/n] "
+  echo -n "Would you like to do that now?"
```

Comment:
It's been DRYed up [here](https://github.com/Originate/git-town/blob/aw-prompt-default/features/step_definitions/town_steps.rb#L61-L65).